OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -292,16 +312,6 @@ protected function getResponseContent(ResponseInterface $rawResponse)
         return $response;
     }
 
-    /**
-     * Generate a non-guessable nonce value.
-     *
-     * @return string
-     */
-    protected function generateNonce()
```

Comment:
Just FYI: Removing this was a hard BC break...

Reply:
Sorry @dmaicher, I didn't realize that it will cause BC. @XWB @stloyd Does it make sense to re-add it as a proxy method maybe?